tests/src/test/java/alluxio/client/cli/fs/command/DistributedLoadCommandTest.java

```diff
@@ -250,4 +249,45 @@ public void loadDirWithPassiveCacheInBatch() throws IOException, AlluxioExceptio
       Assert.assertEquals(100, status.getInMemoryPercentage());
     }
   }
+
+  @Test
+  public void loadDirWithCorrectCount() throws IOException, AlluxioException {
+    FileSystemShell fsShell = new FileSystemShell(ServerConfiguration.global());
+    FileSystem fs = sResource.get().getClient();
+    int fileSize = 66;
+    for (int i = 0; i < fileSize; i++) {
+      FileSystemTestUtils.createByteFile(fs, "/testCount/testBatchFile" + i, WritePType.THROUGH,
+          10);
+      AlluxioURI uri = new AlluxioURI("/testCount/testBatchFile" + i);
+      URIStatus status = fs.getStatus(uri);
+      Assert.assertNotEquals(100, status.getInMemoryPercentage());
+    }
+    fsShell.run("distributedLoad", "/testCount", "--batch-size", "3");
+    String[] output = mOutput.toString().split("\n");
+    Assert.assertEquals(String.format("Completed count is %s,Failed count is 0.", fileSize),
+        output[output.length - 1]);
+  }
+
+  @Test
+  public void loadDirWithFailure() throws IOException, AlluxioException {
+    FileSystemShell fsShell = new FileSystemShell(ServerConfiguration.global());
+    FileSystem fs = sResource.get().getClient();
+    int fileSize = 20;
+    for (int i = 0; i < fileSize; i++) {
+      FileSystemTestUtils.createByteFile(fs, "/testFailure/testBatchFile" + i, WritePType.THROUGH,
+          10);
+      if (i % 2 == 0) {
+        AlluxioURI uri = new AlluxioURI("/testFailure/testBatchFile" + i);
+        URIStatus fileInfo = fs.getStatus(uri);
+        String path = fileInfo.getFileInfo().getUfsPath();
+        boolean result = new File(path).delete();
+        Assert.assertTrue(result);
+      }
+    }
+    fsShell.run("distributedLoad", "/testFailure");
+    Assert.assertTrue(mOutput.toString().contains(
+        String.format("Completed count is %s,Failed count is %s.\n", fileSize / 2, fileSize / 2)));
+    Assert.assertTrue(mOutput.toString().contains(
+        "./logs/user/distributedLoad_testFailure_failures.csv for full list of failed files."));
+  }
```

Comment:
can we read the csv and make sure the expected failures are in there?

Reply:
added